chore: update command-line.md (#4275)
JetBrains/kotlin-web-site: docs/topics/command-line.md

Every Kotlin release ships with a standalone version of the compiler. You can download the latest version manually or via a package manager.

> Installing the command-line compiler is not an essential step to use Kotlin.
> The common approach is to write Kotlin applications using IDEs or code editors with official Kotlin support,
> such as [IntelliJ IDEA](https://www.jetbrains.com/idea/), [JetBrains Fleet](https://www.jetbrains.com/fleet/),
> or [Android Studio](https://developer.android.com/studio).

> > These provide full Kotlin support right out of the box, no extra components needed.

Reviewer: ```suggestion
> They provide full Kotlin support right out of the box, no extra components needed.
```

I feel that "they" is a clearer word here, "these" more naturally refers to something perceived visually.

2. Unzip the standalone compiler into a directory and optionally add the `bin` directory to the system path.
The `bin` directory contains the scripts needed to compile and run Kotlin on Windows, macOS, and Linux.

> > If you want to use the Kotlin command-line compiler on Windows, we recommend using the manual installation method.

Reviewer: ```suggestion
> If you want to use the Kotlin command-line compiler on Windows, we recommend installing it manually.
```

```

Kotlin provides experimental support for script customization, such as adding external properties,
providing static or dynamic dependencies, and so on.

> Customizations are defined by so-called _Script definitions_ – annotated kotlin classes with the appropriate support code.

Reviewer: Should "Script definitions" be capitalized here? On the page linked it looks like a simple term.

Author: Updated!

> Customizations are defined by so-called _Script definitions_ – annotated kotlin classes with the appropriate support code.

Author: ```suggestion
Customizations are defined by so-called _script definitions_ – annotated kotlin classes with the appropriate support code.
```
